src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="get"
+      class="searchBar searchBar--big"
+      data-qa="big"
+    >
+      <input
+        class="searchBar__input searchBar__input--top"
```

Comment:
The `class` attribute has the value 'searchBar__input searchBar__input--top'. A better naming convention would be 'search-bar__input search-bar__input--top' to improve readability.